### DISCUSSION TITLE
Anyone hearing anything about MEF W151?

### DISCUSSION THREAD

**Discussion by jesmanlangit** [MEF-GIT/MEF-LSO-Sonata-SDK#185] [General]
Anyone hearing anything about MEF W151? As per their website, it will be rolled out on December but haven't read any updates about it.

**Comment by MichalLaczynskiAmartus** [comment]
Hello @jesmanlangit. 
May I first ask what particular site are you referring to?
The MEF 151 project page is here:

The last project update ([S87021_001_MEF W151 project update 4Q2023_Bencheck.pptx](https://wiki.mef.net/download/attachments/228688522/S87021_001_MEF%20W151%20project%20update%204Q2023_Bencheck.pptx?version=1&modificationDate=1695399955000&api=v2)) states that the end of the project (LB) aims for 2Q 2023, right now is undergoing CfC#2.

This is only the requirements project in the DSC. Once finished a schema will be developed to support it. I would not expect delivery for December, rather H1 2024. Keep in mind that product schemas are not strictly bound to the releases of the SDKs
